backend/variants/models/maintenance.py

```
@@ -109,3 +109,80 @@ def refresh_variants_smallvariantsummary():
             except utils.NotSupportedError:
                 with transaction.atomic():
                     cursor.execute("REFRESH MATERIALIZED VIEW variants_smallvariantsummary")
+
+
+def drop_variants_smallvariantsummary():
+    """Drop the ``SmallVariantSummary`` materialized view."""
+
+    with transaction.atomic():
+        with connection.cursor() as cursor:
```

Ruff

Use a single `with` statement with multiple contexts instead of nested `with` statements

Combine `with` statements

(SIM117)